qemu/tests/nbd_unix_connection.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from avocado.utils import process
+
+from provider.nbd_image_export import QemuNBDExportImage
+
+
+def run(test, params, env):
+    """
+    1) create a image with qemu-img
+    2) Export the image with qemu-nbd(type=unix)
+    3) Stop to export nbd image
+    4) Repeat connect nbd image by nbdsh cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def run_nbd_connect_cmd(cmd):
+        result = process.run(cmd, timeout=5, ignore_status=True, shell=True)
+        if result.exit_status not in {0, -15}:
```

Comment:
I went through the product bug, it looks all nbdsh should succeed, so here why we check the return code -15?

Reply:
Thanks, as discussed, just to use -15 as accepted exit code.